content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
```

Comment:
@bitwiseman That's a good suggestion, but I think we could tighten it even further.

`In the <<jenkinsfile#, previous section>>, we implemented a Jenkinsfile to check into source control. This section covers the concept of
+*Multibranch* Pipelines, which build on that `Jenkinsfile` to provide...`

@rtyler I'll go through this again today or this weekend and make active voice suggestions. Go unfry your brain!